srcpkgs/kubernetes-helm/template

```diff
@@ -39,11 +39,26 @@ pre_build() {
 do_build() {
 	cd golib/src/k8s.io/helm
 	TARGETS="linux/${_go_target_arch}" make bootstrap build-cross APP="..."
+
+	_dist/linux-${_go_target_arch}/helm completion bash > completion.bash
```

Comment:
can't you use GOARCH directly instead of defining your own variable.

Reply:
I can't, actually. Helm has a very weird build mechanism where it handles cross compiling on its own. Having $GOARCH set to anything but the host arch causes problems for the build, so I need to define my own variable to hold the value.

When I submitted the package, I attempted to make this work with `build-style=go` but it turned out to be even uglier than this. (#1151)

Reply:
Ok I understand. Can you at least change the sub shell sourcing into smth like this, since the current way is is printing an useless error message when running the update check
```
case "$XBPS_TARGET_MACHINE" in
	aarch64*) _go_target_arch=arm64;;
	armv*) _go_target_arch=arm;;;
	i686*) _go_target_arch=386;;
	x86_64*) _go_target_arch=amd64;;
esac
```

Reply:
I will do that. I wasn't sure whether it was better to live with the annoying message, or to duplicate code.